project/Dependencies.scala

```diff
@@ -122,6 +125,7 @@ object Dependencies {
   def excludeJakartaActivationApi = ExclusionRule("jakarta.activation", "jakarta.activation-api")
   def excludeJakartaXmlBindApi = ExclusionRule("jakarta.xml.bind", "jakarta.xml.bind-api")
   def excludeJakarta(m: ModuleID): ModuleID = m.excludeAll(excludeJakartaActivationApi, excludeJakartaXmlBindApi)
+  def terraCommonLibExcludeList(m: ModuleID): ModuleID = m.excludeAll(excludePostgresql, excludeSnakeyaml)
```

Comment:
why not add these two exclusions to the already-existing `tclExclusions`?

Reply:
Ah, I didn't put together "tcl" => "terra common libs". I thought it was some reference to either the language, or some specific library 🤦 . +1 to combining